pkg/sql/schemachanger/screl/attr.go

```diff
@@ -172,6 +176,7 @@ var elementSchemaOptions = []rel.SchemaOption{
 		rel.EntityAttr(ConstraintID, "ConstraintID"),
 		rel.EntityAttr(ReferencedSequenceIDs, "UsesSequenceIDs"),
 		rel.EntityAttr(ReferencedTypeIDs, "UsesTypeIDs"),
+		rel.EntityAttr(IndexIDToValidate, "IndexIDToValidate"),
```

Comment:
You don't actually need this new attribute type, instead `rel.EntityAttr(IndexID, "IndexIDToValidate"),` will do.